app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,79 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign = nil)
+    @assign = {
+      :new     => assign[:new],
+      :current => assign[:current]
+    }
+
+    build = false unless @assign[:new][:assign_to]
+    super(name, type, sandbox, build)
+  end
+
+  def override(node, object, _pid, _options)
+    identifier = (object.name.presence || object.description)
+    if object.kind_of?(MiddlewareServer)
+      identifier += "-" + object.hostname
+    end
+
+    if @assign[:new][:assign_to].ends_with?("-tags")
+      icon = "tag.png"
+    else
+      if @assign[:new][:assign_to] == "ext_management_system"
+        icon = "vendor-#{object.image_name}.png"
+      elsif @assign[:new][:assign_to] == "resource_pool"
+        icon = object.vapp ? "vapp.png" : "resource_pool.png"
+      else
+        icon = "#{@assign[:new][:assign_to]}.png"
+      end
+    end
+
+    node[:title] = identifier
+    node[:image] = ActionController::Base.helpers.image_path("100/#{icon}")
```

Comment:
> "100/#{icon}"

Don't do that please :). Ever. Use the full path, otherwise these are impossible to find.

But do check recent PRs by @epwinchell , especially https://github.com/ManageIQ/manageiq-ui-classic/pull/154 for the new icon values. (For vendor-*, you'll still need to set `node[:image]` but to `svg/vendor-#{object.image_name}.svg`; for resource pool, it will be setting `node[:icon]` to `pficon-resource-pool`, regardless of `vapp`, etc.)